### PR TITLE
Colm/ops 2147

### PR DESCRIPTION
**Pull request by cavster**

### CODE REVIEW

File: app/service/WorkingDaysService.scala [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+@Singleton
+class WorkingDaysService @Inject() () {
+
+  implicit val hols: BankHolidaySet = BankHolidays()
+
+  def addWorkingDays(date: LocalDate, days: Int): LocalDate = {
+    val joda = turnJavaTimeToJoda(date)
+    val jodaLocalDate: time.LocalDate = joda.plusWorkingDays(days)
+    LocalDate.now().withYear(jodaLocalDate.getYear).withDayOfMonth(jodaLocalDate.getDayOfMonth).withMonth(jodaLocalDate.getMonthOfYear)
```
Comment (pawelpanasewicz) [inline app/service/WorkingDaysService.scala]: How you gonna test it? Please use `LocalDate.now(clock)` version of this method. Inject `clock` instance.

File: app/service/WorkingDaysService.scala [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import java.time.LocalDate
+
+import javax.inject.{Inject, Singleton}
+import org.joda.time
```
Comment (pawelpanasewicz) [inline app/service/WorkingDaysService.scala]: Please use `java.time` instead of `joda`

Comment (cavster) [reply app/service/WorkingDaysService.scala]: The hmrc libary takes in joda

File: app/service/WorkingDaysService.scala (hunk 1 above)
Comment (cavster) [reply app/service/WorkingDaysService.scala]: There is already tests q

Comment (pawelpanasewicz) [reply app/service/WorkingDaysService.scala]: mkay

File: app/service/WorkingDaysService.scala (hunk 2 above)
Comment (pawelpanasewicz) [reply app/service/WorkingDaysService.scala]: :( ok, let's use joda then

**Pull request merged**